File: REST_T4/RESTApp/src/java/Entities/accommodationDesire.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Entities;

/**
 *
 * @author Samuel
 */
public class accommodationDesire {
    public int code;
    public String destination;
    public String beginDate;
    public String endDate;
    public int numberPerson;
    public int age[];
    public int cardNumber;
    public String dateValid;
    public int secretNumber;
    public int parts;
    
    public accommodationDesire(int code, String destination, String beginDate, 
             String endDate, int numberPerson, int age[], int cardNumber, 
             String dateValid, int secretNumber, int parts) {
        this.code           = code;
        this.destination    = destination;
        this.beginDate      = beginDate;
        this.endDate        = endDate;
        this.numberPerson   = numberPerson;
        this.age            = age;
        this.cardNumber     = cardNumber;
        this.dateValid      = dateValid;
        this.secretNumber   = secretNumber;
        this.parts          = parts;
    }
}    

File: client_python/accommodationDesire.py
# -*- coding: utf-8 -*-    

###############################################################################
# Class is used for easily built a JSON output to send it for the Java WEB 
# Service server.
###############################################################################

class accommodationDesire():
    def __init__(self, code, beginDate, 
             endDate, numberPerson, age, cardNumber, 
             dateValid, secretNumber, parts):
        self.code           = code;
        self.beginDate      = beginDate;
        self.endDate        = endDate;
        self.numberPerson   = numberPerson;
        self.age            = age;
        self.cardNumber     = cardNumber;
        self.dateValid      = dateValid;
        self.secretNumber   = secretNumber;
        self.parts          = parts;
        
 
    def to_JSON(self):
        return {"code":self.code, "beginDate":self.beginDate, 
            "endDate":self.endDate, "numberPerson":self.numberPerson, "age":self.age,
            "cardNumber":self.cardNumber, "dateValid":self.dateValid, 
            "secretNumber":self.secretNumber, "parts":self.parts}
